<?xml version="1.0"?>
<launch>

    <param name="use_sim_time" value="true"/>

    <node pkg="stefmap_ros" type="stefmap_node_offline.py" name="stefmap_node" output="screen">
    	<param name="grid_size" value="1"/>  
		<param name="x_min" value="-50" />
    	<param name="x_max" value="50" />
    	<param name="y_min" value="-50" />
    	<param name="y_max" value="50" />
    	<param name="num_bins" value="8" />
    	<param name="frame_id" value="map" />
    </node>

	<node pkg="fremenarray" type="fremenarray" name="fremenarray" output="screen">
    </node>

</launch>